### ISSUE TITLE
Axes object error when passing valid axes object

### ISSUE THREAD

**Issue by sco1** [sco1/coordinate2normalized#5]
The following error is received when calling `coordinate2normalized` with a valid axes object in R2014a:

```
>> coordinate2normalized(axes, [1 2], [1 2])

Error using coordinate2normalized>checkinputs (line 25)
First input to function must be a MATLAB Axes object, not double

Error in coordinate2normalized (line 2)
checkinputs(axishandle, x, y);
```

 This input executes without error in R2015b

**Comment by sco1** [comment]
Input check utlizes `isa` to check whether the `axishandle` input is an `Axes`:

```
~isa(axishandle, 'matlab.graphics.axis.Axes')
```

This works for MATLAB R2014b and newer with the implementation of HG2, as [graphics handles are now objects](http://www.mathworks.com/help/matlab/graphics_transition/graphics-handles-are-now-objects-not-doubles.html). In releases prior to R2014b, `axishandle` is a `double` so this error will be thrown even with a valid axes handle.

**Comment by sco1** [comment]
Fix for `norm2coord` implemented in 22b071d, 42da430

**Comment by sco1** [comment]
Fix for `coord2norm` implemented in 2ef8177
